bin/dump-data.pl

```diff
@@ -181,7 +183,7 @@ sub db_quote {
 # and dump mood info
 print "Dumping moods.dat\n";
 open( F, ">$ENV{'LJHOME'}/bin/upgrading/moods.dat" ) or die;
-$sth = $dbh->prepare("SELECT moodid, mood, parentmood FROM moods ORDER BY moodid");
+$sth = $dbh->prepare("SELECT moodid, mood, parentmood, weight FROM moods ORDER BY moodid");
```

Comment:
don't know when this became a thing, but moods.dat contains it so we should dump it.